packages/shipjs/src/step/release/createGitHubRelease.js

```diff
@@ -2,7 +2,6 @@ import path from 'path';
 import fs from 'fs';
 import globby from 'globby';
 import { Octokit } from '@octokit/rest';
-import mime from 'mime-types';
```

Comment:
From a quick GitHub search, it looks like `mime-types` was only used for this feature, so it looks like we can remove the dependency.

Reply:
[7f728d6](https://github.com/algolia/shipjs/pull/1000/commits/7f728d60434a3f9eb1952a08b393a863d657a38e)